src/applications/personalization/profile/tests/e2e/notification-settings/missing-contact-info-mobile.cypress.spec.js

```diff
@@ -34,14 +34,7 @@ describe('Notification Settings For Mobile Phone', () => {
         user.data.attributes.vet360ContactInformation.mobilePhone = null;
         cy.login(user);
         cy.visit(PROFILE_PATHS.NOTIFICATION_SETTINGS);
-        cy.get('va-loading-indicator')
-          .should('exist')
-          .then($container => {
-            cy.wrap($container)
-              .shadow()
-              .findByRole('progressbar', { name: /loading/i })
-              .should('exist');
-          });
+        cy.get('va-loading-indicator').should('exist');
```

Comment:
This was flakey in CI/CD and the secondary check inside the `cy.get` is not needed, so I removed it and kept the outer check to make sure the loading indicator is still being shown.